File: Frontend/Frontend/src/app/demo/api/Models/ProveedoresViewModel.ts
export class ProveedoresViewModel {
    prove_Id!: number;
    prove_Marca!: string;
    prove_ContactoPrimerNombre!: string;
    prove_ContactoSegundoNombre!: string;
    prove_ContactoPrimerApellido!: string;
    prove_ContactoSegundoApellido!: string;
    munic_Id!: string;
    prove_Direccion!: string;
    prove_Telefono!: string;
    prove_Correo!: string;
    prove_Notas!: string;
    prove_UsuarioCreacion!: number;
    prove_FechaCreacion!: Date;
    prove_UsuarioModificacion?: number | null;
    prove_FechaModificacion?: Date | null;
    prove_Estado?: boolean | null;
    munic_Descripcion?: string;
    usuarioCreacion?: string;
    usuarioModificacion?: string;
}

File: Backend/Sistema_Larach/Sistema_Larach.Common/Models/ProveedoresViewModel.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sistema_Larach.Common.Models
{
  public  class ProveedoresViewModel
    {
        public int Prove_Id { get; set; }
        public string Prove_Marca { get; set; }
        public string Prove_ContactoPrimerNombre { get; set; }
        public string Prove_ContactoSegundoNombre { get; set; }
        public string Prove_ContactoPrimerApellido { get; set; }
        public string Prove_ContactoSegundoApellido { get; set; }
        public string Munic_Id { get; set; }
        public string Prove_Direccion { get; set; }
        public string Prove_Telefono { get; set; }
        public string Prove_Correo { get; set; }
        public string Prove_Notas { get; set; }
        public int Prove_UsuarioCreacion { get; set; }
        public DateTime Prove_FechaCreacion { get; set; }
        public int? Prove_UsuarioModificacion { get; set; }
        public DateTime? Prove_FechaModificacion { get; set; }
        public bool? Prove_Estado { get; set; }
        [NotMapped]
        public string Munic_Descripcion { get; set; }
        [NotMapped]
        public string UsuarioCreacion { get; set; }
        [NotMapped]
        public string UsuarioModificacion { get; set; }
    }
}
